price_security/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+  :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+  :alt: License: AGPL-3
+
+==============
+Price Security
+==============
+
+Creates a new permission to restrict the users that can modify the prices
+of the products.
+
+Asociate to each user a list of pricelist and the correspoding discounts they
+can apply to sale orders and invoices.
+
+Allow the posibility to mark products so that anyone can modify their price in
+a sale order.
+
+
+NEW: sequence field on pricelist and payment term (you can only assigne pricelests or terms of lower priority than partner default one)
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Just install this module.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. No needed configuration.
```

Comment:
Pondría: (revisar nombre del grupo y lugar para setear descuentos, no recuerdo como se llaman)
To configure this module, you need to:

#. Set 'Restrict Prices' for users you want to restrict
#. For the same users, configure discounts range on "asdas" users tab